core/build.gradle

```diff
@@ -135,8 +135,8 @@ dependencies {
     testCompile group: 'org.skyscreamer', name: 'jsonassert', version: '1.5.0'
     testCompile group: 'org.assertj', name: 'assertj-core', version: '3.13.2'
 
-    compileOnly group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.22.3'
-    testCompile group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.22.3', {
+    compileOnly group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.22.5'
```

Comment:
Why did we stop at 2.22.5? There are even newer ones released: https://mvnrepository.com/artifact/com.google.cloud/google-cloud-storage

Reply:
Ah okay, I normally try to go for newer so we have longer between when we need to fix it next :)

Reply:
That makes sense, will change it to the newest